Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -266,8 +266,9 @@ TextureCache::TCacheEntryBase* TextureCache::Load(const u32 stage)
 	const int texformat = tex.texImage0[id].format;
 	const u32 tlutaddr = tex.texTlut[id].tmem_offset << 9;
 	const u32 tlutfmt = tex.texTlut[id].tlut_format;
-	u32 tex_levels = (tex.texMode1[id].max_lod + 0xf) / 0x10 + 1;
-	const bool use_mipmaps = (tex.texMode0[id].min_filter & 3) != 0 && tex_levels > 0;
+	const bool mipmap_min_filter = (tex.texMode0[id].min_filter & 3) != 0;
+	u32 tex_levels = mipmap_min_filter ? ((tex.texMode1[id].max_lod + 0xf) / 0x10 + 1) : 1;
+	const bool use_mipmaps = mipmap_min_filter && tex_levels > 0;
```

Comment:
Damn, this bug is here before the custom texture rewrite, so textures with 1 level and mipmap filter settings did get the "_m" flag. Are you fine with just removing this check and keeping the "_m" flag within the custom texture names if the filter is enabled? For the texture cache itself, this doesn't matter.